### PR TITLE
adding proper guava library + excluding it from all other dependencies

### PR DESCRIPTION
**Pull request by savitru**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -203,6 +207,25 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-yarn-registry</artifactId>
+            <version>3.1.1.3.1.0.0-78</version>
```
Comment (phaneesh) [inline pom.xml]: Use stack versio variable :)

File: pom.xml [hunk 2]
```diff
@@ -203,6 +207,25 @@
...
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>19.0</version>
```
Comment (phaneesh) [inline pom.xml]: Declare guava.version as property and use it here

File: pom.xml (hunk 1 above)
Comment (savitru) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (savitru) [reply pom.xml]: done

**Review by phaneesh — APPROVED** [review]

**Pull request merged**